cds-snc/report-a-cybercrime
# save i18n.locale to database

Fixes #1334 

# Description

Save the user's language to the database

To test:
1) Go to review app, get to confirmation page
2) open dev tools / console (command-option-i on Macs, then find console tab)
3) click submit. Console should display "Submitting data: " along with the data object. Find the language element.
4) click the browser's back arrow, change language, and resubmit. language in data object should be different.

# Checklist:
